qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/datamodel/resolvers/ValueResolverInfo.java

```diff
@@ -106,13 +108,31 @@ public String getSourceType() {
 
 	/**
 	 * Set the java source type.
-	 * 
+	 *
 	 * @param sourceType the java source type
 	 */
 	public void setSourceType(String sourceType) {
 		this.sourceType = sourceType;
 	}
 
+	/**
+	 * Returns the java target and null otherwise.
+	 *
+	 * @return the java targe and null otherwise.
+	 */
+	public String getTarget() {

```

Comment:
Please remove target in value resolver you don't need (for the moment). We will see if we need it when you will take care of target for static fields / methods.